extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -155,6 +159,9 @@ export class AmpSidebar extends AMP.BaseElement {
       element.setAttribute('side', this.side_);
     }
 
+    const width = element.getAttribute('width');
```

Comment:
Sorry but I'm lacking context behind this change. Was this a new feature request? Is this backwards compatible (since this is an important style, and the width attribute might have been used to different effect i.e. no effect before)?

Reply:
I think I changed this since amp-list had some layout issue inside sidebar without a width attribute; but testing it now I can't really reproduce the problem. I can add this change to the design doc or remove it before release. Regardless, this should be backwards compatible since if the user sets sidebar width via custom css it would override the width set by css variable.

Reply:
I am more concerned about users accidentally setting the `width` attribute (due to autogenerated code) and assuming that it is a no-op. Since `amp-sidebar` is layout no-display, I think it will ignore the width and height attributes, which are still valid AMP but have no effect. So if someone accidentally sets `width=0` and expects it to do nothing, this code will now make their sidebar not show up. 

Because this is in a version-bumped instance of sidebar, this won't hurt our existing users. But it could make the upgrade path more difficult. If this code is not necessary, I would recommend removing it.

Reply:
That makes sense; I'll remove the change.